tools/test/go.mod

```diff
@@ -1,10 +1,12 @@
 module github.com/stackrox/stackrox/tools/test
 
-go 1.21
+go 1.22.0
 
-toolchain go1.21.7
+toolchain go1.22.1
 
 require (
 	github.com/jstemmer/go-junit-report/v2 v2.1.0
-	github.com/stackrox/image-prefetcher/deploy v0.1.0
+	github.com/stackrox/image-prefetcher/deploy v0.1.1-0.20240521100631-ccbc2e464f14
```

Comment:
Should we wait with merging it for:
- https://github.com/stackrox/image-prefetcher/pull/12